pkg/usersettings/types.go

```diff
@@ -0,0 +1,20 @@
+package usersettings
+
+type ResourceData struct {
```

Comment:
ResourceData sounds a bit odd. How about `UserSettingData` or maybe `UserSettingMeta` ?

Reply:
Had the same feeling but no better name. :smile: Change it to `UserSettingMeta`